crates/geoip/src/block.rs

```diff
@@ -0,0 +1,90 @@
+use {crate::resolver::GeoIpResolver, bitflags::bitflags, std::net::IpAddr};
+
+#[cfg(feature = "middleware")]
+pub mod middleware;
+
+bitflags! {
```

Comment:
why switch from enum? `bitflags` aren't that good for public API and we won't be using bit operations on it
If it's a memory layout optimization, then `#[repr(u8)]` should do?

Reply:
Ah, wait, it's intended to be use with bitwise ops? ok then, nvm

Reply:
That was the idea for when we need to target specific regions etc. For now though using bitflags does not affect the public API in any way, except for maybe simplifying a bit.